### PR TITLE
Integration specs for views

### PR DESCRIPTION
**Pull request by elmar8287**
Use Capybara to write integration tests for each view in your project.

for:

- Login page
- User index page
- User show page
- User post index page
- Post show page

elmar

### CODE REVIEW

File: spec/views/posts/index_posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  scenario "click on post and redirect to that post's show page." do
+    @post = Post.create(title: 'Bla bla bla', text: 'Bla bla bla', author_id: @user1.id)
+    find("a[href='#{user_path(@user1.id)}']").click
+    find("a[href='#{user_posts_path(@user1.id)}']").click
+    find("a[href='#{user_post_path(@user1.id, @post.id)}']").click
+    expect(page).to have_current_path(user_post_path(@user1.id, @post.id))
+  end
+  # rubocop:enable Metrics/BlockLength
+end
```
Comment (DeliceLydia) [inline spec/views/posts/index_posts_spec.rb]: - [ ] Please fix your views errors all are failing, this is an example
view

v2

v1

v0

File: spec/views/posts/show_posts_spec.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    @post = Post.create(title: 'Bla bla Bla', text: 'Bla bla Bla', author_id: @user1.id)
+    @comment1 = Comment.create(text: 'Bla bla Bla', author_id: @user1.id,
+                               post_id: @post.id)
+    find("a[href='#{user_path(@user1.id)}']").click
+    find("a[href='#{user_posts_path(@user1.id)}']").click
+    find("a[href='#{user_post_path(@user1.id, @post.id)}']").click
+    expect(page).to have_content @comment1.text
+  end
+  # rubocop:enable Metrics/BlockLength
+end
```
Comment (DeliceLydia) [inline spec/views/posts/show_posts_spec.rb]: - [ ] 43 tests are failing please fix them
v

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! my major highlights for your projects :
- Commits messages are descriptive
- Shout out to the good README



## Required Changes ♻️
_Check the comments under the review._



### Have fun while coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @elmar8287 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by elmar8287** [comment]
Thanks for the review! 👍

**Pull request merged**